apps/payments/next/app/[locale]/subscriptions/manage/page.tsx

```diff
@@ -49,6 +50,9 @@ export default async function Manage({
       ? l10n.getLocalizedMonthYearString(expMonth, expYear, locale)
       : undefined;
 
+  const MANAGE_LINK_CSS =
```

Comment:
Could we add a variant in `ButtonVariant` instead? Open to naming suggestions -ButtonVariant.SecondaryManage?